std/zig/render.zig

```diff
@@ -1668,15 +1634,15 @@ fn renderExpression(
 
             var it = asm_node.clobbers.iterator(0);
             while (true) {
-                const clobber_token = it.next().?;
+                const clobber_node = it.next().?.*;
```

Comment:
Can there be 0 items?

Reply:
Nothing changed here. Length of `clobbers` is already checked here: https://github.com/ziglang/zig/blob/0d629421c97f5be7a89102c0ae57916418e71243/std/zig/render.zig#L1617-L1620

But now that you bring my attention here, it would be nicer if the section below matched the `if-payload` style of the other asm parts, instead of `if (it.peek() == null)`: https://github.com/ziglang/zig/blob/0d629421c97f5be7a89102c0ae57916418e71243/std/zig/render.zig#L1608